tests/api-tests/db-enable-logging.test.ts

```diff
@@ -27,11 +27,12 @@ test(
       expect(logs).toEqual([
         [
           'prisma:query',
-          dbProvider === 'sqlite'
+          (dbProvider === 'sqlite'
             ? 'SELECT `main`.`User`.`id`, `main`.`User`.`name` FROM `main`.`User` WHERE 1=1 LIMIT ? OFFSET ?'
             : dbProvider === 'mysql'
             ? 'SELECT `test_db`.`User`.`id`, `test_db`.`User`.`name` FROM `test_db`.`User` WHERE 1=1'
-            : 'SELECT "public"."User"."id", "public"."User"."name" FROM "public"."User" WHERE 1=1 OFFSET $1',
+            : 'SELECT "public"."User"."id", "public"."User"."name" FROM "public"."User" WHERE 1=1 OFFSET $1') +
+            ' /* traceparent=00-00-00-00 */',
```

Comment:
What is this?

Reply:
This is just a test about enabling Prisma's logging, i assume the added bit is related to https://www.prisma.io/docs/concepts/components/prisma-client/opentelemetry-tracing